app/Builders/Hierarchy/Hierarchy.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Builders\Hierarchy;
+
+class Hierarchy
```

Comment:
```suggestion
class HierarchyBuilder
```

cs-fixer i zapusk qilin

Reply:
+